overlord/devicestate/devicemgr.go

```diff
@@ -142,6 +142,9 @@ func Manager(s *state.State, hookManager *hookstate.HookManager, runner *state.T
 
 	runner.AddBlocked(gadgetUpdateBlocked)
 
+	// add FDE hook support to boot
```

Comment:
we usually put this kind of initialisation in a delayedCrossMgrInit functions or method I suppose

Reply:
Thanks, fixed